src/common/storage/backends/sciserver-files/Client.js

```diff
@@ -48,60 +44,56 @@ define([
         return this.createDataInfo(metadata);
     };
 
-    SciServerFiles.prototype.deleteDir = async function(dirname) {
+    SciServerFiles.prototype.deleteDir = async function (dirname) {
         const url = `data/Storage/${this.volume}/${dirname}`;
         const opts = {method: 'DELETE'};
         return await this.fetch(url, opts);
     };
 
-    SciServerFiles.prototype.deleteFile = async function(dataInfo) {
+    SciServerFiles.prototype.deleteFile = async function (dataInfo) {
         const {volume, filename} = dataInfo.data;
         const url = `data/Storage/${volume}/${filename}`;
         const opts = {method: 'DELETE'};
         return await this.fetch(url, opts);
     };
 
-    SciServerFiles.prototype.getMetadata = async function(dataInfo) {
+    SciServerFiles.prototype.getMetadata = async function (dataInfo) {
         const metadata = {size: dataInfo.data.size};
         return metadata;
     };
 
-    SciServerFiles.prototype.getDownloadURL = async function(dataInfo) {
+    SciServerFiles.prototype.getDownloadURL = async function (dataInfo) {
         const {data} = dataInfo;
         return data.url;
     };
 
-    SciServerFiles.prototype._stat = async function(volume, path) {
+    SciServerFiles.prototype._stat = async function (volume, path) {
         const fullpath = volume + '/' + path;
         const url = `1/metadata/sandbox/${fullpath}?list=True&path=${fullpath}`;
-        const headers = new Headers();
-        headers.append('Content-Type', 'application/xml');
-        const response = await this.fetch(url);
+        const headers = {
```

Comment:
I would remove the use of `headers` as it was dead code previously (and there were no known issues resulting from it) rather than unnecessarily using it.

Reply:
Or I would ignore it in this PR and just leave it to be addressed in a different issue